README.rst

```diff
@@ -92,26 +93,38 @@ We assume you may have enough storage space on your PC.
 How to add your code to code base
 ==============================================================
 
+The code to be added to the code base must have a git repository.
+
 You have to write a .lcopy file and put it to lcopy.d directory.
 See lcopy.d/linux.lcopy as an example::
 
     REPO=https://github.com/torvalds/linux
     ALIGNMENT=v4.20
     LANGUAGES=C,LdScript,Asm,Kconfig,DTS
 
+`codebase` commands loads a .lcopy file with source built-in command. In
+a .lcopy file must define `REPO`, `ALIGNMENT`, and `LANGUAGES`.
+
 `REPO` specifies a git repository.  `ALIGNMENT` is a tag put on the
-git repository.  `ALIGNMENT` allows users of codebase to get the same
-source tree. `LANGUAGES` is a comma separated language list.
+git repository.  `ALIGNMENT` allows users of `codebase` to get the same
+source tree. Specify a git tag or commit as `ALIGNMENT`. A git branch
+is not acceptable. `LANGUAGES` is a comma separated language list.
 
 
-How to add your profile to preset list
+How to add a new profile
 ==============================================================
 
-You have to write a .ctags file and put it to profile.d directory.
-A line started from "# @" is used as a description for the profile.
-You may wan to use `--options-maybe` to extend profile without
-modifying existing .ctags files.
+Add a .ctags file under profile.d directory.
+In the .ctags file, a line started from "# @" is used as a
```

Comment:
Before this sentence you should describe that a line started from "#" is a comment.

And now I found a ".tags file" means a "option file" described in FILES sections of  man ctags(1).
If we describe that this is a option file, a reader can understand that he/she can use comment lines starting '#' and the "--options" option can be used.